granary/reddit.py

```diff
@@ -137,14 +137,20 @@ def praw_to_object(self, thing, type):
     obj['url'] = self.BASE_URL + thing.permalink
 
     if type == 'submission':
-      obj['content'] = getattr(thing, 'title', None)
+      obj['name'] = getattr(thing, 'title', None)
+      obj['content'] = getattr(thing, 'selftext', None)
       obj['objectType'] = 'note'
       obj['tags'] = [
           {'objectType': 'article',
            'url': t,
            'displayName': t,
            } for t in util.extract_links(getattr(thing, 'selftext', None))
         ]
+
+      if getattr(thing, 'url', None):
+          obj['objectType'] = 'bookmark'
+          obj['targetUrl'] = getattr(thing, 'url')
```

Comment:
minor nit: this can be just `thing.url`. not strictly necessary for this PR though.